package/network/services/dropbear/Config.in

```diff
@@ -67,6 +67,34 @@ config DROPBEAR_CHACHA20POLY1305
 
 		Increases binary size by about 4 kB (MIPS).
 
+config DROPBEAR_U2F
```

Comment:
Please explain in the commit description why you split this.

Reply:
Done.